graphbolt/include/graphbolt/csc_sampling_graph.h

```diff
@@ -64,6 +64,26 @@ class CSCSamplingGraph : public torch::CustomClassHolder {
       const torch::optional<torch::Tensor>& node_type_offset,
       const torch::optional<torch::Tensor>& type_per_edge);
 
+  /**
+   * @brief Performs neighbor sampling for a given set of seed nodes taking edge
+   * type into account, where each edge type has a specified pick number.
+   *
+   * @param seed_nodes The tensor containing the seed nodes.
+   * @param fanouts The tensor containing the number of neighbors to sample per
+   * edge type.
+   * @param replace Boolean indicating if sampling is done with replacement.
+   * @param return_eids Boolean indicating if edge IDs are required to return,
+   * which is usually used when edge features are required.
+   * @param probs Optional tensor containing probabilities for sampling. Dtype
+   * should be bool or float.
+   *
+   * @return A pointer to a SampledSubgraph object representing the sampled
+   * subgraph.
+   */
+  c10::intrusive_ptr<SampledSubgraph> SampleEtypeNeighbors(
+      torch::Tensor seed_nodes, torch::Tensor fanouts, bool replace,
+      bool return_eids, const torch::optional<torch::Tensor>& probs);
```

Comment:
Why we use const quote for probs, but not for seed_nodes and fanouts?

Reply:
Should add 'const &' for `Tensor` Object. BTW， `Tensor`  content is still mutable even at 'const' descriptor.